config/HH_bbWW/Run3_2022/samples.yaml

```diff
@@ -1,444 +1,620 @@
+############## Start Resonant Signal samples)##############
 ############## Signals(SL,S=0)##############
 GluGlutoRadiontoHHto2B2Vto2B2JLNu_M-250:
-  sampleType: GluGluToRadion
+  sampleType: GluGluToRadion_SL
```

Comment:
i thought it may useful if we want to run only SL on DL not both every time. Now I realised that code in non-config area also needs to change to accommodate this. So far now i will revert this granularity, and we will see if we really need this during next iterations.